web/profiles/joinup/joinup.behat.inc

```diff
@@ -2672,4 +2672,99 @@ JS;
     Assert::assertNotEmpty($element);
   }
 
+  /**
+   * Asserts that the content absolute urls exist as links.
+   *
+   * @param \Behat\Gherkin\Node\TableNode $titles
+   *   The titles of the nodes.
+   *
+   * @throws \Exception
+   *   Thrown if any link does not exist in the page.
+   *
+   * @Given I should see the absolute urls of the following content entities:
+   */
+  public function assertContentAbsoluteUrlsAsLinkInPage(TableNode $titles): void {
+    foreach ($titles->getRows() as $title) {
+      $this->assertEntityUrlInSitemap('node', $title[0]);
+    }
+  }
+
+  /**
+   * Asserts that the rdf entity absolute urls exist as links.
+   *
+   * @param \Behat\Gherkin\Node\TableNode $titles
+   *   The titles of the rdf entities.
+   *
+   * @throws \Exception
+   *   Thrown if any link does not exist in the page.
+   *
+   * @Given I should see the absolute urls of the following RDF entities:
+   */
+  public function assertRdfEntityAbsoluteUrlsAsLinkInPage(TableNode $titles): void {
+    foreach ($titles->getRows() as $title) {
+      $this->assertEntityUrlInSitemap('rdf_entity', $title[0]);
+    }
+  }
+
+  /**
+   * Asserts that the content absolute urls do not exist as links.
+   *
+   * @param \Behat\Gherkin\Node\TableNode $titles
+   *   The titles of the nodes.
+   *
+   * @throws \Exception
+   *   Thrown if at least a link exists in the page.
+   *
+   * @Given I should not see the absolute urls of the following content entities:
+   */
+  public function assertEmptyContentAbsoluteUrlsAsLinkInPage(TableNode $titles): void {
+    foreach ($titles->getRows() as $title) {
+      $this->assertEntityUrlInSitemap('node', $title[0], FALSE);
+    }
+  }
+
+  /**
+   * Asserts that the absolute url of an rdf entity exists as a link.
+   *
+   * @param \Behat\Gherkin\Node\TableNode $titles
+   *   The titles of the rdf entities.
+   *
+   * @throws \Exception
+   *   Thrown if the link does not exist in the page.
+   *
+   * @Given I should not see the absolute urls of the following RDF entities:
+   */
+  public function assertEmptyRdfEntityAbsoluteUrlsAsLinkInPage(TableNode $titles): void {
+    foreach ($titles->getRows() as $title) {
+      $this->assertEntityUrlInSitemap('rdf_entity', $title[0], FALSE);
+    }
+  }
+
+  /**
+   * Asserts a url in the sitemap response.
+   *
+   * @param string $entity_type_id
+   *   The entity type id.
+   * @param string $entity_id
```

Comment:
This is named wrongly. It's not the entity ID but the entity label. `$entity_label`?